### PR TITLE
DCOS_OSS-714: Go To Bottom-button not hiding at the bottom

### PR DESCRIPTION
**Pull request by mlunoe**
---
~~⚠️ Depends on this PR: https://github.com/dcos/dcos-ui/pull/1954~~ (merged)

---

This PR fixes the 'Go To Bottom' button not hiding when at the bottom.

Please set this in your config dev:
```js
mesos: {
  'logging-strategy': 'logrotate'
},
```

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

For reviewers: View diff between PRs [here](https://github.com/dcos/dcos-ui/compare/mlunoe/feature/DCOS-14010-add-container-to-pick-logging-component-and-api...mlunoe/bug-fix/DCOS_OSS-714-is-at-bottom-button-not-hiding-at-the-bottom?expand=1)

### CODE REVIEW

**Comment by rcorral** [comment]
I changed the base branch so that this is easier to review

File: plugins/services/src/js/components/LogView.js [hunk 1]
```diff
@@ -163,7 +163,9 @@ class LogView extends React.Component {
   }
 
   checkIfAwayFromBottom(container) {
-    const isAtBottom = container.offsetHeight + container.scrollTop
+    // The 200px is to give the checker a little bit of breating room to
+    // determine when we are at the bottom
+    const isAtBottom = container.offsetHeight + container.scrollTop + 200
```
Comment (jfurrow) [inline plugins/services/src/js/components/LogView.js]: #nit Perhaps we should move this value to a `const` at the top of this file, and name it something that indicates it's the distance-from-bottom threshold.

**Comment by rcorral** [comment]
@mlunoe I tested this out but it wasn't working for me when using `lograte`.
I added a commit to fix it for lograte too.

**Comment by jfurrow** [comment]
Seems to be working well for me!

**Comment by MatApple** [comment]
@orlandohohmeier @wavesoft @nLight this is working for me also but please check this too. Haven't added my label as I don't want it merged without more eyes. I suggest we thoroughly test the logging feature branch before we merge to master.

**Comment by nLight** [comment]
The button hides, though autoscroll doesn't kick in is it ok?

**Comment by orlandohohmeier** [comment]
Aside from the *autoscroll* looks good to me. I'd say let's open an issue for the autscoll feature and merge this one.

**Comment by orlandohohmeier** [comment]
I've filed an issue for the autoscroll issue. https://jira.mesosphere.com/browse/DCOS_OSS-734

**Pull request merged**